consul/client.go

```diff
@@ -215,7 +229,13 @@ func (c *Client) Encrypted() bool {
 
 // lanEventHandler is used to handle events from the lan Serf cluster
 func (c *Client) lanEventHandler() {
+	var numQueuedEvents int
 	for {
+		numQueuedEvents = len(c.eventCh)
+		if numQueuedEvents > serfEventBacklogWarning {
```

Comment:
If this becomes noisy, definitely, and its something that I imagine will see some attention very soon. *grin*  In the interim this seems like one of the things that we want to plop into the logs.  The only time this should be seen is when the rate of processing serf events is slower than the number of events being added, which results in stalls for serf.  Right now we don't know how common this is or isn't and this seemed like a good way to start observing the situation.  Before 0.7 I expect we might have something different in place.